app/views/sessions/failure_to_proof.erb

```diff
@@ -0,0 +1,16 @@
+<%= render 'shared/alert_error', alert_heading: t('alerts.failure_to_proof.heading'), alert_text: t('alerts.failure_to_proof.text') %>
+
+<div>
+  <h3>What you can do:</h3>
```

Comment:
shouldn't this be h1 since it's the only header on the page?

Reply:
The `shared/alert_error` has an `h3` because of its use on other pages

Reply:
But the page needs an `h1` and `axe_clean` doesn't seem to complain about the out of order headings